front/components/navigation/Navigation.tsx

```diff
@@ -55,70 +76,72 @@ export function Navigation({
+              <Button
+                variant="ghost"
+                icon={MenuIcon}
+                onClick={() => setSidebarOpen(true)}
+              />
+            </SheetTrigger>
+          </div>
+          <SheetContent
+            side="left"
+            className="flex w-full max-w-xs flex-1 bg-muted-background dark:bg-muted-background-night"
           >
-            {navChildren && navChildren}
-          </NavigationSidebar>
-        </SheetContent>
-      </Sheet>
-
-      {/*Desktop sidebar*/}
-      <div
-        className={cn(
-          "transition-width hidden flex-none overflow-hidden duration-150 ease-out lg:flex lg:flex-col",
-          isNavigationBarOpen ? "w-80" : "w-0"
-        )}
-      >
-        <div className="hidden flex-1 bg-muted-background dark:bg-muted-background-night lg:inset-y-0 lg:z-0 lg:flex lg:w-80 lg:flex-col">
-          <NavigationSidebar
-            owner={owner}
-            subscription={subscription}
-            subNavigation={subNavigation}
-            user={user}
+            <SheetHeader className="bg-muted-background p-0" hideButton={true}>
+              <VisuallyHidden>
+                <SheetTitle className="hidden" />
+              </VisuallyHidden>
+            </SheetHeader>
+            <NavigationSidebar
+              subscription={subscription}
+              owner={owner}
+              subNavigation={subNavigation}
+              user={user}
+              isMobile={true}
+            >
+              {navChildren && navChildren}
+            </NavigationSidebar>
+          </SheetContent>
+        </Sheet>
+      ) : (
+        <>
+          <div
+            className={cn(
+              "transition-width flex-none overflow-hidden duration-150 ease-out lg:flex lg:flex-col",
+              isNavigationBarOpen ? "w-80" : "w-0"
+            )}
           >
-            {navChildren && navChildren}
-          </NavigationSidebar>
-        </div>
-      </div>
+            <div className="flex-1 bg-muted-background dark:bg-muted-background-night lg:inset-y-0 lg:z-0 lg:flex lg:w-80 lg:flex-col">
+              <NavigationSidebar
+                owner={owner}
+                subscription={subscription}
+                subNavigation={subNavigation}
+                user={user}
+              >
+                {navChildren && navChildren}
+              </NavigationSidebar>
+            </div>
+          </div>
 
-      <div
-        className={classNames(
-          "fixed z-40 hidden lg:top-1/2 lg:flex",
-          isNavigationBarOpen ? "lg:ml-80" : ""
-        )}
-      >
-        <ToggleNavigationSidebarButton
-          isNavigationBarOpened={isNavigationBarOpen}
-          toggleNavigationBarVisibility={(navigationBar) => {
-            setNavigationBarOpen(navigationBar);
-          }}
-        />
-      </div>
+          <div
+            className={classNames(
+              "fixed z-40 hidden lg:top-1/2 lg:flex",
+              isNavigationBarOpen ? "lg:ml-80" : ""
```

Comment:
I guess we can remove the condition right? Since we are only mounting the component for desktop